k8s/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: block-explorer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: block-explorer
+    spec:
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
+      containers:
+      - name: block-explorer
+        image: planetariumtest.azurecr.io/<IMAGE_NAME>
```

Comment:
Should this be changed by hand, or is it a k8s's grammar?

Reply:
It should be changed by hand.